portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,21 +26,37 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  String[] rankings = {"VOSS", "FIJI", "SMART WATER", "CRYSTAL GEYSER"};
-  ArrayList<String> water_rankings = new ArrayList<String>();
-
-  public DataServlet() {
-      water_rankings.add("VOSS");
-      water_rankings.add("FIJI");
-      water_rankings.add("SMART WATER");
-      water_rankings.add("CRYSTAL GEYSER");
+  ArrayList<String> questions = new ArrayList<String>();
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "question-input", "");
+
+    questions.add(text);
+
+    // Respond with the result.
+    response.sendRedirect("index.html");
```

Comment:
I'm not sure about this part. If your intend is to load the page and show the new entries, there are better ways.